### ISSUE TITLE
Infinity?

### ISSUE THREAD

**Issue by kaiminari** [mikebryant/ac-nh-turnip-prices#401]
The graph was pretty accurate this week up until I input my prices for this morning and got “infinity”.
![9F9EE1BE-5363-4B75-A01B-4541A7CDCBFD](https://user-images.githubusercontent.com/66129802/83275364-e6038700-a183-11ea-9c63-b69c756eb430.png)
![891BBCC6-B66C-4166-969F-5D1975721784](https://user-images.githubusercontent.com/66129802/83275380-e9970e00-a183-11ea-9ce8-bed427f4ad9b.jpeg)

**Comment by Lysoun** [comment]
Could you please give the permalink to your prediction?

**Comment by kaiminari** [comment]
> Could you please give the permalink to your prediction?

https://turnipprophet.io/?prices=104.68..60...130.103.192.38...&pattern=2

**Comment by shanep2300** [comment]
I have the same issue for this week. First time it's happened since we started using this tool for about 2ish months.

https://turnipprophet.io/?prices=101.54.58.54.51.46.......&first=false&pattern=0&acnh.guide

**Comment by anaagnello** [comment]
It happened to me today as well!! 

https://turnipprophet.io?prices=92.63.60.56.52.49.136......&pattern=3

**Comment by itscifer** [comment]
Happened to me today, what a bug, first week I try it.
![FireShot Capture 033 - Animal Crossing - Turnip Prophet - turnipprophet io](https://user-images.githubusercontent.com/58343774/83793940-86ced680-a69d-11ea-98b7-a3a4f2a79873.png)

**Comment by punchbuggyy** [comment]
Same here! First week trying, was accurate up until this afternoon.
![FEB8C6B0-7A65-4E8E-851B-0DBF6847912C](https://user-images.githubusercontent.com/66450315/83825417-21bfb480-a68e-11ea-92a4-6f17098bf8d1.png)

Permalink: https://turnipprophet.io/?prices=95.64.59.56.52.47.43.131.61....

**Comment by julesghouls** [comment]
I was super confused when this happened to me too haha, but I thought it might've somehow been caused by skipping Tuesday (since the store was closed for renovations).

https://turnipprophet.io?prices=95.95.68...64.58.113.....

**Comment by vaidashwin** [comment]
I also encountered this issue this week: https://turnipprophet.io/?prices=109..60.45.41.35.102.122.152.154.153..&pattern=1

**Comment by joshvro** [comment]
Yeah, I have the same issue. Here’s a per a link to mine: https://turnipprophet.io/?prices=91.47.125.101.97.64.118.128.131.61.56.46.111

**Comment by bellexandria** [comment]
I also have the same issue :(

https://turnipprophet.io?prices=94.104.123..99..62.106..30...

<img width="1249" alt="Screen Shot 2020-06-19 at 11 26 54 AM" src="https://user-images.githubusercontent.com/39638114/85150295-5b60f580-b220-11ea-9296-8594797b5a49.png">

The only thing I see in the dev console is 

```
(index):336 [PWA] Service Worker already found, skipping register
service-worker.js:47 [PWA] Network request failed and no cache.no-match
service-worker.js:1 Uncaught (in promise) no-match
```

...and a bunch of 404s relating to i18n.

Maybe more debugging can be added leading up to https://github.com/mikebryant/ac-nh-turnip-prices/blob/bba8007a472204ae83b19305e1cc2899c83509ab/service-worker.js#L63 ?

**Comment by fallgaming** [comment]
I also received this bug today. https://turnipprophet.io?prices=110..93.136.115.64.......&pattern=0

**Comment by hbjosemaria** [comment]
Same result: https://turnipprophet.io?prices=97.83.77.150.73........&pattern=0

![Screenshot_20200707_150056](https://user-images.githubusercontent.com/13330079/86792816-c143de80-c062-11ea-9a8b-c04f8c1dbd09.jpg)

**Comment by stephanied930-boop** [comment]
I’m also having this issue

<img width="585" height="1266" alt="Image" src="https://github.com/user-attachments/assets/e17bf609-2acc-449c-ac66-fa457376ec9e" />
